articles/tutorials/qft-tutorial.md

```diff
@@ -466,22 +598,24 @@ The final operation code should look like:
             DumpMachine();
 
             for(i in IndexRange(qs)) {
-                set r w/= i <- M(qs[i]);
+                set resultArray w/= i <- M(qs[i]);
             }
 
             Message("After measurement: ");
             DumpMachine();
 
             ResetAll(qs);
         }
-        return r;
+        Message("Post-QFT measurement results [qubit0, qubit1, qubit2]: ");
+        return resultArray;
     }
 }
 ```
 
-Additionally, update your host program to process the returned array:
+<!-- Additionally, update your host program to process the returned array: -->
```

Comment:
Similarly here, it'd be good to drop this rather than keeping it as a comment, I think.